src/sandbox/components/Grid.svelte

```diff
@@ -13,33 +15,63 @@
   export let onkeydown;
   export let transparent = false;
   export let overflowHidden = false;
+  export let isEditMode;
 
   const thisSheet = getContext('store');
   const { nRows, nCols } = $thisSheet;
   const rowHeight = 20, columnWidth = 100;
 
   let focused = false;
   let editorSession = null;
+  let isFormula = false;
 
-  let container = null, editor = null;  // refs
+  let container = null, editor = null;
   const dispatch = createEventDispatcher();
 
+  let isNodeInserted = false;
+  let isArrowMode = false;
+  let insertRange = null;
+  let highlight = selection;
+  let arrowRow, arrowCol;
+
+  $: if (editorSession !== null) isFormula = detectFormula(editorSession)
+
+  $: isEditMode = (editor !== null);
+  $: if (!isEditMode) {
+    insertRange = null;
+    isArrowMode = false;
+  }
+  $: if (isEditMode) highlight = selection;
```

Comment:
"else" would be better

Reply:
Sorry, but I'm not quite sure I understand well the meaning of this comment. Could you please explain it?

Reply:
You have 2 lines:
`$: if (!isEditMode)`and
`$: if (isEditMode) `

and then
`$: if (isArrowMode)`and
`$: if (!isArrowMode)`
`if () else` would do, no?

Reply:
Yes, that would be the way better. Done in upcoming commit!